bitnami/contour/templates/service.yaml

```diff
@@ -21,11 +24,11 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "contour.fullname" . }}-envoy
-  labels: {{- include "contour.labels" . | nindent 4 }}
+  name: {{ include "common.names.fullname" . }}-envoy
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
     app.kubernetes.io/component: envoy
     {{- if .Values.envoy.service.labels }}
-    {{- include "contour.tplValue" (dict "value" .Values.envoy.service.labels "context" $) | nindent 4 }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.envoy.service.labels "context" $) | nindent 4 }}
     {{- end }}
   annotations:
     {{- if (ne (get .Values.envoy.service.annotations "service.beta.kubernetes.io/aws-load-balancer-type") "nlb") }}
```

Comment:
See comment here: https://github.com/bitnami/charts/pull/2961/files#r484070226

> While working on #3381 and testing my changes with helm 2.14 and 2.16 I get this error:
> 
> ```
> helm2 lint .
> 
> ==> Linting .
> [ERROR] templates/: parse error in "contour/templates/service.yaml": template: contour/templates/service.yaml:34: function "get" not defined
> 
> Error: 1 chart(s) linted, 1 chart(s) failed
> ```
> 
> I realized that this is a function introduced in [Sprig v3](https://github.com/Masterminds/sprig/releases/tag/v3.0.0): [Masterminds/sprig#197](https://github.com/Masterminds/sprig/pull/197).
> 
> Since bitnami needs helm v2 compat, I think we should try to solve this using an available function.

Reply:
Sure!! good catch!